README.md

```diff
@@ -30,13 +30,13 @@ zone.  This role has a trust relationship with the users account.
 | aws.route53resourcechange | ~> 3.38 |
 | terraform | n/a |
 
-## Modules ##
+## Modules
 
 | Name | Source | Version |
 |------|--------|---------|
 | read\_terraform\_state | github.com/cisagov/terraform-state-read-role-tf-module | n/a |
 
-## Resources ##
+## Resources
```

Comment:
Unfortunately, `terraform-docs` disagress with our Markdown linter rule for headings, so this must be changed back to:
```suggestion
## Resources ##
```